.travis.yml

```diff
@@ -26,8 +26,10 @@ script:
   - cargo fmt -- --check &&
     cargo test --verbose --release
   - if [ "${TRAVIS_OS_NAME}" = linux ]; then
-      cargo clippy --verbose --release &&
-      cargo clippy --verbose --release --profile=test
```

Comment:
It's probably failing due to terminating `;` - can you check that first pls before making this change ?

Also if going for this eventually, see if you need `set -x` or something for subshell to report error or commands being run etc.,